apps/betterangels/src/app/(private-screens)/add-note/ProvidedServices/ProvidedCheckbox.tsx

```diff
@@ -100,7 +100,7 @@ export default function ProvidedCheckbox(props: IProvidedCheckboxProps) {
           ...services,
           {
             id,
-            customService: '',
```

Comment:
Hi @Davit-BetterAngels 
I found a few things we need to modify. Please take a look at the attachments and see the descriptions below.

1. Modify Metro Life Tap to Metro LIFE Tap (capital case on LIFE)
2. Social Security Card Replacement - Let's show this in 2 lines since it's pushing the checkbox 
3. Remove "Stabilize"
4. Remove "Replacement" from Unemployment Certification Replacement 
5. Remove "Storage"
6. Remove "Transport"

Thanks and let me know if you have any questions. 

<img src="https://github.com/user-attachments/assets/e0ad43ea-e47f-489d-b19d-06b43639f9bd" width="360" />

<img src="https://github.com/user-attachments/assets/25624f8d-2779-441b-a75b-62c85ef16246" width="360" />